examples/classification/test_MultinomialCNB.py

```diff
@@ -0,0 +1,123 @@
+"""
+================================
+Comparing Complement Naive Bayes to standard Naive Bayes.
+================================
+
+An example showing how MultinomialCNB outperforms MultinomialNB
+on imbalanced text classification tasks.
+
+"""
+print(__doc__)
+import numpy as np
+import random
+
+from nltk.corpus import reuters
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+from sklearn.metrics import precision_score, recall_score
+from sklearn.naive_bayes import MultinomialCNB, MultinomialNB
+
+# Some setup code taken from "Classifying Reuters-21578 collection with Python"
+# on https://miguelmalvarez.com.
+random.seed(2010)
+documents = reuters.fileids()
+
+train_docs_id = list(filter(lambda doc: doc.startswith("train"),
+                            documents))
+test_docs_id = list(filter(lambda doc: doc.startswith("test"),
+                           documents))
+
+train_docs = [reuters.raw(doc_id) for doc_id in train_docs_id]
+test_docs = [reuters.raw(doc_id) for doc_id in test_docs_id]
+
+train_labels = [random.choice(reuters.categories(doc_id))
+                for doc_id in train_docs_id]
+test_labels = [random.choice(reuters.categories(doc_id))
+               for doc_id in test_docs_id]
+
+count_vect = CountVectorizer()
+X_train_counts = count_vect.fit_transform(train_docs)
+X_test_counts = count_vect.transform(test_docs)
+
+accuracies = {}
+precisions = {}
+recalls = {}
+
+for model in [MultinomialNB, MultinomialCNB]:
+    accuracies[model] = {}
+    precisions[model] = {}
+    recalls[model] = {}
+
+    clf = model().fit(X_train_counts, train_labels)
```

Comment:
I would have pass to instance probably instead of the class. It will depends of the dictionary key remark.